include/mbedtls/ecdsa.h

```diff
@@ -1,9 +1,10 @@
 /**
  * \file ecdsa.h
  *
- * \brief The Elliptic Curve Digital Signature Algorithm (ECDSA).
+ * \brief This file contains ECDSA definitions and functions.
```

Comment:
I feel like this is a good place to have the full name with an abbreviation in the parentheses.

Reply:
Disagree. The brief description refers to the content of the file. The detailed description contains all other information, including any standard description or reference. Having the full name in the brief makes it harder to read with comprehension.